src/main/java/racingcargame/ResultView.java

```diff
@@ -0,0 +1,30 @@
+package racingcargame;
+
+import java.util.List;
+
+public class ResultView {
+    private static final String MOVING_LINE = "-";
+
+    public void showResultView(List<RacingCar> cars) {
+       for (RacingCar car : cars) {
+           System.out.println(convertDistanceToLine(car.getDistance()));
```

Comment:
아래와 같이 가독성 좋게 표현해보시는건 어떨까용?
```suggestion
           String line = convertDistanceToLine(car.getDistance());
           System.out.println(line);
```